nixos/modules/services/matrix/appservice-irc.nix

```diff
@@ -20,7 +20,7 @@ let
   } ''
     # The schema is given as yaml, we need to convert it to json
     python -c 'import json; import yaml; import sys; json.dump(yaml.safe_load(sys.stdin), sys.stdout)' \
-      < ${pkg}/lib/node_modules/matrix-appservice-irc/config.schema.yml \
+      < ${pkg}/config.schema.yml \
```

Comment:
Would be simpler to use `remarshal`, which is also widely used throughout `pkgs.formats`.

```
remarshal --if yaml --of json -i ${pkg}/config.schema.yml -o config.schema.json
```

Reply:
I didn't want to change that code too much since I had no idea what and why it does it.
But sure.

Reply:
changed.

`nixosTests.matrix-appservice-irc` did run fine, so I guess I didn't introduce more bugs :D